src/state/store.tsx

```diff
@@ -1,29 +1,54 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { createStore, applyMiddleware } from 'redux';
 import thunkMiddleware from 'redux-thunk';
+import { persistStore, persistReducer } from 'redux-persist';
+import { createWhitelistFilter, createBlacklistFilter } from 'redux-persist-transform-filter';
+import storage from 'redux-persist/lib/storage';
+import autoMergeLevel2 from 'redux-persist/lib/stateReconciler/autoMergeLevel2';
 import rootReducer from './rootReducer';
 
+const persistConfig = {
+  key: 'root',
+  storage,
+  stateReconciler: autoMergeLevel2,
```

Comment:
Because we are saving only a part of state. If I use the default settings it will only load whatever is saved and deletes the rest of the state -- which is not what we want.

https://github.com/rt2zz/redux-persist#state-reconciler